src/wp-includes/class-wp-scripts.php

```diff
@@ -464,7 +519,19 @@ public function print_inline_script( $handle, $position = 'after', $display = tr
 		$output = trim( implode( "\n", $output ), "\n" );
 
 		if ( $display ) {
-			printf( "<script%s id='%s-js-%s'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), esc_attr( $position ), $output );
+			if ( 'after-non-standalone' === $position ) {
+				$script_output = "<script%1\$s id='%2\$s-js-after' data-wp-executes-after='%2\$s'>\n%4\$s\n</script>\n";
```

Comment:
This is a small bug, we must not use the placeholder for `$this->type_attr` here, as that would result in a regular `script` tag.
```suggestion
				$script_output = "<script type='text/template' id='%2\$s-js-after' data-wp-executes-after='%2\$s'>\n%4\$s\n</script>\n";
```

Reply:
cf. https://github.com/10up/wordpress-develop/pull/58/files#r1185533189